client/src/components/Dialog/InterestQuizDialog/QuizDialog.js

```diff
@@ -53,8 +54,23 @@ class QuizDialog extends React.Component {
         this.setState({ selected });
     }
 
-    handleConfirm () {
-        console.log(this.state.selected);
+    async handleConfirm () {
+        const username = this.props.location.pathname.split('/')[2];
```

Comment:
This is split is incorrect, but you actually don't even need it. You can find the url params at props.match.params

Reply:
so this would be this.props.match.params.username

Reply:
right, thanks